pageflow.gemspec

```diff
@@ -21,7 +21,11 @@ Gem::Specification.new do |s|
   s.add_dependency 'rails', '~> 5.2.0'
 
   # Framework for admin interface
-  s.add_dependency 'activeadmin', '~> 1.3.0'
+  if RUBY_VERSION >= '2.4'
+    s.add_dependency 'activeadmin', '~> 2.5.0'
+  else
+    s.add_dependency 'activeadmin', '~> 2.0.0'
+  end
```

Comment:
The gemspec is evaluated at gem release time. So this has no effect when the gem is used in another project and makes the declared dependencies depend on the Ruby version on the machine of the developer who released the gem.

There is no reason to _require_ an Active Admin version above 2.5 in the Ruby 2.4 case if our gem works with Active Admin 2.0 anyway. Looking at the changes in the PR, I'd suspect that we could even go with `s.add_dependency 'activeadmin', ['>= 1.3.0', '< 3']`.

Reply:
yes it works